src/main/java/com/microsoft/azure/vmagent/AzureVMCloudBaseRetentionStrategy.java

```diff
@@ -11,6 +14,8 @@
         implements Serializable {
 
     private static final transient long LAPSE_START_JENKINS = TimeUnit.MINUTES.toMillis(3);
+    private static final int FREE_SPACE_THRESHOLD_MB = 100;
```

Comment:
Does this override or affect the free space threshold configuration via Jenkins/computer/configure?